matplotlib/cfg_notests.patch

```diff
@@ -0,0 +1,13 @@
+diff --git setup.cfg.template setup.cfg.template
```

Comment:
I think it is referenced in meta.yaml (on line 12) -- or am I missing something?